examples/table_creation.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# coding=utf-8
+"""Examples of using the cmd2 table creation API"""
+import functools
+import sys
+from typing import Any, List
+
+from cmd2 import ansi
+from cmd2.table_creator import AlternatingTable, BorderedTable, Column, HorizontalAlignment, SimpleTable
+
+
+class DollarFormatter:
+    """Example class to show that any object type can be passed as data to TableCreator and converted to a string"""
+    def __init__(self, val: float) -> None:
+        self.val = val
+
+    def __str__(self) -> str:
+        """Returns the value in dollar currency form (e.g. $100.22)"""
+        return "${:,.2f}".format(self.val)
+
+
+# Text styles used in the data
+bold_yellow = functools.partial(ansi.style, fg=ansi.fg.bright_yellow, bold=True)
+blue = functools.partial(ansi.style, fg=ansi.fg.bright_blue)
+green = functools.partial(ansi.style, fg=ansi.fg.green)
+
+# Table Columns
+columns: List[Column] = list()
```

Comment:
This type syntax doesn't work with python 3.5. Since we officially support that version, recommend changing so the example works in all supported versions.

Reply:
I actually argued this same point in PR #900. However, it was decided, and I agreed, that supporting Python 3.5 in examples is no longer required.